README.md

```diff
@@ -1,10 +1,10 @@
 # Teckyl: An MLIR frontend for Tensor Operations
 
-Teckyl is released under the the Apache License v2.0 with LLVM
-Exceptions. For details see the file LICENSE.
+Teckyl is released under the Apache License v2.0 with LLVM
+Exceptions. For details, see the file LICENSE.
 
-This project is based on llvm-project. See the llvm-project
-subdirectory for details about the licsensing.
+This project is based on the llvm-project. See the llvm-project
```

Comment:
I would agree if this were "LLVM project" instead of "llvm-project". I would prefer to omit the article here, since llvm-project can be considered a proper name.